core/unbound_method.rbs

```diff
@@ -213,7 +263,7 @@ class UnboundMethod
   #
   #     (1..3).method(:map).owner #=> Enumerable
   #
-  def owner: () -> Module
+  def owner: () -> (Class | Module)
```

Comment:
Do we need `Class` here? `Class` is a subclass of `Module`.

Reply:
Not technically, but I thought it was a bit clearer, as most new rubyists don't know `Class` is an instance of `Module`.